framework/java/android/net/wifi/WifiConfiguration.java

```diff
@@ -1811,16 +1812,23 @@ public boolean isOpenNetwork() {
      */
     public static final int RANDOMIZATION_AUTO = 3;
 
+    /**
+     * @hide
```

Comment:
Remove this line, make it the same as other constants.

Reply:
okay,